### DISCUSSION TITLE
[FEATURE] adding  new challenge

### DISCUSSION THREAD

**Discussion by mohitahlawat2001** [lennis-dev/hackit#13] [Ideas]
### Is there an existing discussion for this?

- [X] I have searched the existing discussions

### Describe your idea

Want to add new security challenge, let me know files , which I have to change, inorder to make adding new challenge success

**Comment by lennis-dev** [comment]
Nice that you want to participate, do you have a name and a short description?
Like this: https://github.com/lennis-dev/hackit/blob/c2717800e499e55357afde95a07471aff253103f/challenges/easy-beginning/index.php#L10-L11

and can you tell me what the challenge is about?

**Comment by lennis-dev** [comment]
No response
